resources/assets/js/jsvalidation.js

```diff
@@ -117,6 +117,10 @@ laravelValidation = {
             data = {};
             data[ element.name ] = value;
             data['_jsvalidation']= attribute;
+            var formMethod = $(validator.currentForm).attr('method');
+            if($(validator.currentForm).find('input[name="_method"]').length) {
+                formMethod = $(validator.currentForm).find('input[name="_method"]');
```

Comment:
Would you please give an example of an acceptance criteria for a line length? I think this one is pretty clean.